vignettes/main_functions.Rmd

```diff
@@ -0,0 +1,460 @@
+
+```{r mnrCoding setup, message=FALSE}
+prepDat <- automateDataPreparation(inputList = inputList, 
+    datList = inputDat, readSpss = FALSE, checkData=FALSE, 
+    mergeData = TRUE, recodeData=TRUE, aggregateData=FALSE, 
+    scoreData= FALSE, writeSpss=FALSE, verbose = TRUE)
+prepDat2 <- mergeData("ID", list(prepDat, inputList$rotation))
+```
+
+```{r mnrCoding example}
+mnrDat <- mnrCoding(dat = prepDat2, pid = "ID", 
+    booklets = inputList$booklets, blocks = inputList$blocks, 
+    rotation.id = "booklet", breaks = c(1, 2), 
+    subunits = inputList$subunits, nMbi = 2, mbiCode = "mbi", 
+    mnrCode = "mnr", invalidCodes = c("mbd", "mir", "mci"), 
+    verbose = TRUE)
+```
+
+Type `?mnrCoding` into your console to learn more. 
+
+## Wrapper
+
+Instead of using all the individual functions as described above, you can also use the wrapper function `automateDataPreparation`, which contains most of the described functions. Here is an overview over the arguments that call the corresponding function.
+
+```{r wrapper overview, echo=FALSE}
+readspss <- c("`readSpss`", "readSpss()")
+checkdata <- c("`checkData`", "checkData()")
+mergedata <- c("`mergeData`", "mergeData()")
+recodedata <- c("`recodeData`", "recodeData()")
+recodemnr <- c("`recodemnr`", "mnrCoding()")
+aggregatedata <- c("`aggregateData`", "aggregateData()")
+scoredata <- c("`scoreData`", "scoreData()")
+collapsemissings <- c("`collapseMissings`", "collapseMissings()")
+writespss <- c("`writeSpss`", "writeSpss()")
+
+wrap <- rbind.data.frame(readspss, checkdata, mergedata, recodedata, recodemnr,
+                         aggregatedata, scoredata, collapsemissings, writespss)
+names(wrap) <- c("Argument", "Corresponding Function")
+kable(wrap)
+```
+
+But there are many more arguments that may need to consider. For more information see `?automateDataPreraration`. 
+
+Here is an example use case and its output. `automateDataPreparation` returns a data frame resulting from the final data preparation step.
+
+```{r wrapper example}
+data(inputList)
+data(inputDat)
+preparedData <- automateDataPreparation(inputList = inputList,
+    datList = inputDat,  path = getwd(),
+    readSpss = FALSE, checkData = TRUE,  mergeData = TRUE,
+    recodeData = TRUE, recodeMnr = TRUE, breaks = c(1,2),
+    aggregateData = TRUE, scoreData = TRUE,
+    writeSpss = FALSE, verbose = TRUE)
+```
+
+
+## Exporting Data
+
+After the data preparation is done, you can prepare the data sets for exporting to then use them in the packages `eatModel` and `eatGADS`.
+
+### Collapsing Data
+
+The last step is to recode character missings to numerical types like 0 or NA with `collapseMissings()`. All variables need to be converted to `numeric`, as well, so you can pass the data set to `eatModel` afterwards. 
+
+You need the data frame that is returned by `recodeData`, which we called `datRec` earlier (`dat`). 
```

Comment:
no, it works with datRec, but its main purpose is to finish the scored dataset as it comes out of scoreData or automateDataPreparation